exercises/concept/cache/cache.go

```diff
@@ -0,0 +1,9 @@
+package cache
+
+func GetValueFromFastestLayer(cacheLayers []Cache, key string) (value string, err error) {
```

Comment:
Please remove the names from the return values. Those should only be used when really needed as they can lead to mistakes easily.